felix/bpf/nat/connecttime.go

```diff
@@ -72,50 +63,64 @@ func RemoveConnectTimeLoadBalancer(cgroupv2 string) error {
 		return nil
 	}
 
-	cgroupPath, err := ensureCgroupPath(cgroupv2)
+	bpfMount, err := utils.MaybeMountBPFfs()
 	if err != nil {
-		return errors.Wrap(err, "failed to set-up cgroupv2")
-	}
-
-	cmd := exec.Command("bpftool", "-j", "-p", "cgroup", "show", cgroupPath)
-	log.WithField("args", cmd.Args).Info("Running bpftool to look up programs attached to cgroup")
-	out, err := cmd.Output()
-	if err != nil || strings.TrimSpace(string(out)) == "" {
-		log.WithError(err).WithField("output", string(out)).Info(
-			"Failed to list BPF programs.  Assuming not supported/nothing to clean up.")
-		return err
+		return errors.Wrap(err, "Failed to mount bpffs")
```

Comment:
```suggestion
		return fmt.Errorf("failed to mount bpffs: %w", err)
```

Since we are at it, lets ditch the old way of wrapping